### PR TITLE
Get more understandable error message on deep structure

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: jsonschema_gentypes/validate.py [hunk 1]
```diff
@@ -111,21 +111,29 @@ def format_error(error: jsonschema.exceptions.ValidationError) -> str:
...
-        return (
-            f" - {position} "
-            f'({".".join([str(i) for i in error.path] if error.path else "/")}): '
-            f"{error.message} (rule: "
-            f'{".".join([str(i) for i in error.schema_path] if error.schema_path else "/")})'
-        )
-
-    return sorted([format_error(e) for e in validator.iter_errors(data)]), data
+        if error.context:
+            result = []
```
Comment (ger-benjamin) [inline jsonschema_gentypes/validate.py:122]: small things, but `results` would be better (with a `s` at the end)
Same at line 133

File: tests/test_validate.py [hunk 2]
```diff
@@ -63,9 +63,40 @@ def test_validate_yaml():
...
+                            "type": "object",
+                            "properties": {
+                                "test": {"type": "string"},
+                            },
+                        }
+                    },
+                },
+            },
+        },
+    )
```
Comment (ger-benjamin) [inline tests/test_validate.py:97]: strange indentation no ?

**Review by ger-benjamin — APPROVED** [review]

File: tests/test_validate.py (hunk 2 above)
Comment (sbrunner) [reply tests/test_validate.py:97]: For me, it's correct, It's done by Black...

File: jsonschema_gentypes/validate.py (hunk 1 above)
Comment (sbrunner) [reply jsonschema_gentypes/validate.py:122]: +1 Thanks

**Pull request merged**